ui/Modal/Search.svelte

```diff
@@ -53,7 +57,9 @@
     }
   };
   const follow = () => {
-    requestProject(value);
+    if ($urnValidation.status === ValidationStatus.Success) {
+      requestProject(value);
```

Comment:
It’s probably worth adding a test for invalid URNs, either in this PR or as a follow-up

Reply:
045faac2fd0ed352b4a2e8d9c0b65f3e12655356